myuw/dao/instructor.py

```diff
@@ -0,0 +1,32 @@
+from myuw.models import SeenInstructor
+from uw_sws.term import get_term_before
```

Comment:
there's no request object in the context, to the term.py refactor seemed only marginally useful